lib/transformers/rawContent.js

```diff
@@ -16,7 +16,7 @@ module.exports = ({schemaKey, schema, options} = {}) => {
     );
 
     // parse out dependencies from content
-    const rxComponents = /(?<=govuk)\S+?(?=\s*\()/g;
+    const rxComponents = /(?<=govuk)\S+?(?=\s*\({)/g;
```

Comment:
There isn't an a false test in place for this regex on line 25  of this file. 
There wasn't one before so I am calling it out. 
 ```
 const rxComponents = /(?<=govuk)\S+?(?=\s*\()/g;
    const componentTypes = opts.content.match(rxComponents);

    let output = opts.content;

    // Get dependencies for component
    if (componentTypes) {
```
If you think it isn't necessary that's fine.

Reply:
There was a regex test it was just named obscurely. I've added another as a negative test when no nunjuck instruction is added and renamed the current one. 
![image](https://user-images.githubusercontent.com/119854428/231160480-c075d6b7-409d-47c9-9295-f0162172f696.png)